kalite/main/api_views.py

```diff
@@ -10,9 +10,33 @@
 from fle_utils.internet.webcache import backend_cache_page
 
 from kalite.topic_tools import get_topic_tree
+from kalite.topic_tools.content_recommendation import get_resume_recommendations, get_next_recommendations, get_explore_recommendations
+from kalite.facility.models import FacilityUser
 
 @api_handle_error_with_json
 @backend_cache_page
 def topic_tree(request, channel):
     parent = request.GET.get("parent")
     return JsonResponse(get_topic_tree(channel=channel, language=request.language, parent=parent))
+
+@api_handle_error_with_json
+def content_recommender(request):
```

Comment:
Yes, @sialan had valiantly wrestled with it, but as I was cleaning it up today, it become so much easier just to plomp everything into a normal django view.